x-pack/test/stack_functional_integration/apps/apm/apm_smoke_test.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function ({ getService, getPageObjects }) {
+  describe('APM smoke test', function ampsmokeTest() {
+    const browser = getService('browser');
+    const testSubjects = getService('testSubjects');
+    const PageObjects = getPageObjects(['common', 'settings', 'timePicker']);
```

Comment:
Looks like we are not using the settings page object in this test. 
So, we should drop the `'settings',` bit.